src/core/lib/channel/channelz.h

```diff
@@ -101,17 +101,24 @@ class CallCountingHelper {
   // testing peer friend.
   friend class testing::CallCountingHelperPeer;
 
-  struct CounterData {
+  struct AtomicCounterData {
     gpr_atm calls_started_ = 0;
```

Comment:
Struct fields don't need the `_` suffix.

Reply:
Done